.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,122 @@
+name: Tag Repository for Release to Staging
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag (YYYY.MM.DD) to push. Defaults to current day"
+        required: false
+      push:
+        description: "Should the tag be pushed to github. (false means dry run)"
+        required: false
+        default: 'false'
```

Comment:
I lean towards pushing being the default?  (otherwise what's the purpose of the action?) with maybe a `dry-run` parameter for testing?